app/components/primer/open_project/tree_view/icon_pair.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Primer
+  module OpenProject
+    class TreeView
+      # A pair of icons for a `TreeView` sub-tree that displays distinct icons when the sub-tree is
+      # expanded and collapsed.
+      #
+      # This component is part of the <%= link_to_component(Primer::OpenProject::TreeView) %> component and should
+      # not be used directly.
+      class IconPair < Primer::Component
+        # The expanded icon.
+        #
+        # @param system_arguments [Hash] The arguments accepted by <%= link_to_component(Primer::OpenProject::TreeView::Icon) %>.
+        renders_one :expanded_icon, lambda { |**system_arguments|
+          Icon.new(**system_arguments)
+        }
+
+        # The collapsed icon.
+        #
+        # @param system_arguments [Hash] The arguments accepted by <%= link_to_component(Primer::OpenProject::TreeView::Icon
```

Comment:
Missing parenthesis and closing `%>`

```suggestion
        # @param system_arguments [Hash] The arguments accepted by <%= link_to_component(Primer::OpenProject::TreeView::Icon) %>.
```